platform/ui/src/components/userPreferencesForm/HotKeysPreferences.js

```diff
@@ -231,16 +231,31 @@ function HotKeyPreferencesRow({
     setInputValue(formatPressedKeys(hotkeys));
   }, [hotkeys]);
 
+  const isSpecialCharacter = str => {
+    return str.length === 1 && str.match(/^[^a-zA-Z0-9]+$/);
```

Comment:
^ - start of word
[^] - Negation (match anything not in set)
+ - Match preceding token

I didn't debug this to see what my value is here, just breaking it down for my own reference.